x/pool/types/keys.go

```diff
@@ -0,0 +1,15 @@
+package types
+
+const (
+	// ModuleName is the module name constant used in many places
+	ModuleName = "cosmos-sdk-pool"
```

Comment:
Don't we have a better name?
I know pool will probably clash with some existing module developed by other chains, but this is quite verbose.

Reply:
want to propose something. I asked if we should switch it to cosmos-sdk-pool but maybe cosmos-pool is enough

Reply:
I'm not too good at names, funds, reserve, or simply community-pool.

I do like cosmos-pool too.